blocks/isi-tray/isi-tray.css

```diff
@@ -72,28 +72,35 @@ main .section.isi-tray-container > div {
 
 .isi-tray.block.gammagard-sd button {
   border-color: var(--gammagard-sd);
```

Comment:
I just realized that these should all be `background-color`. All of these border colors can be removed.